extensions/positron-python/python_files/posit/positron/tests/test_positron_jedilsp.py

```diff
@@ -365,26 +396,42 @@ def test_positron_completion(
     source: str,
     namespace: Dict[str, Any],
     character: Optional[int],
-    expected_labels: Union[List[str], Callable[[List[Optional[str]]], bool]],
+    expected_labels: List[str],
     monkeypatch,
+    tmp_path,
 ) -> None:
-    server = create_server(namespace)
+    # Set the root path to an empty temporary directory so there are no file completions.
+    server = create_server(namespace, root_path=tmp_path)
     text_document = create_text_document(server, TEST_DOCUMENT_URI, source)
 
-    # Patch environment variables for the `os.environ` test.
+    # Patch os.environ so that only the test environment variable's completion is ever present.
     with patch.dict(os.environ, clear=True):
-        monkeypatch.setenv(_environment_variable_key, "")
+        monkeypatch.setenv(TEST_ENVIRONMENT_VARIABLE, "")
 
         completions = _completions(server, text_document, character)
 
+    assert_completion_labels_equals(completions, expected_labels)
+
+
+def test_positron_completion_notebook(tmp_path) -> None:
```

Comment:
I only tested completions and signature help for now, since I included lots more tests in the upstream extension and just wanted something minimal here as kind of a smoke test.